src/lxc/start.c

```diff
@@ -472,7 +472,7 @@ int lxc_set_state(const char *name, struct lxc_handler *handler,
 int lxc_poll(const char *name, struct lxc_handler *handler)
 {
 	int ret;
-	bool has_console = (handler->conf->rootfs.path != NULL);
+	struct lxc_console *console = &handler->conf->console;
```

Comment:
Hm, shouldn't this still check whether `strcmp(handler->conf->console->rootfs.path, "none") != 0`?

Reply:
Fixed